test/descriptive-link-text.md

```diff
@@ -0,0 +1,23 @@
+# Descriptive link text
+
+[Learn about Javascript](https://example.com/javascript/about)
```

Comment:
Please add some variety to the destinations. It should not matter, but one can be just "file.txt" and another can be "/dir/file.txt" or whatever. 10 instances of the same URL is not as interesting.